cfgov/scripts/initial_test_data.py

```diff
@@ -39,7 +39,7 @@ def run():
     site_root = HomePage.objects.filter(title='CFGOV')
     if not site_root:
         root = Page.objects.first()
-        site_root = HomePage(title='CFGOV', slug='home', depth=2, owner=admin_user)
+        site_root = HomePage(title='CFGOV', slug='home-page', depth=2, owner=admin_user)
```

Comment:
data migartions are the standard method of data import for django now and they have to be done at some point. Very likely out of scope but seems related enough to justify.